src/todoItem.js

```diff
@@ -0,0 +1,47 @@
+import { uncheckedSvgImage, checkedSvgImage, deleteSvgImage } from "../assets/icons/svgImg.js";
+
+function createImageElement(svgImage) {
+  const buttonImg = document.createElement('img');
+  buttonImg.src = `data:image/svg+xml,${encodeURIComponent(svgImage)}`;
+  return buttonImg;
+}
+
+function handleToggleCompletion(isCompleted, todoTextSpan, unCheckedButtonImg) {
```

Comment:
함수의 매개변수가 많아지면 객체로 받아주는게 좋아요~ 특히나 자바스크립트에서는 사용하는 쪽에서 순서를 알기가 어려워요.

```suggestion
function handleToggleCompletion({
  isCompleted, 
  todoTextSpan, 
  unCheckedButtonImg
}) {
```

Reply:
파라미터의 지정된 순서가 아니라
객체 구조분해 할당으로 하니까 훨씬 좋은데요?!!